ui/base/ime/input_method_minimal_unittest.cc

```diff
@@ -0,0 +1,83 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "ui/base/ime/input_method_minimal.h"
+
+#include "base/macros.h"
+#include "testing/gtest/include/gtest/gtest.h"
+#include "ui/base/ime/dummy_text_input_client.h"
+#include "ui/base/ime/input_method_delegate.h"
+#include "ui/base/ime/input_method_initializer.h"
+#include "ui/events/event.h"
+
+namespace ui {
+namespace {
+
+class InputMethodDelegateForTesting : public internal::InputMethodDelegate {
+ public:
+  InputMethodDelegateForTesting(bool propagation)
+      : propagation_post_ime_(propagation) {}
+  ~InputMethodDelegateForTesting() override {}
+
+  ui::EventDispatchDetails DispatchKeyEventPostIME(
+      ui::KeyEvent* key_event) override {
+    if (!propagation_post_ime_)
+      key_event->StopPropagation();
+    return ui::EventDispatchDetails();
+  }
+
+ private:
+  bool propagation_post_ime_;
+
+  DISALLOW_COPY_AND_ASSIGN(InputMethodDelegateForTesting);
+};
+
+class InputMethodMinimalTest : public testing::Test {
+ protected:
+  InputMethodMinimalTest()
+      : input_method_minimal_(nullptr), delegate_(nullptr) {}
```

Comment:
prefer to use = default.
https://www.chromium.org/developers/coding-style/cpp-dos-and-donts